lldb/test/API/functionalities/data-formatter/data-formatter-stl/libcxx-simulators/unique_ptr/main.cpp

```diff
@@ -0,0 +1,43 @@
+#include "../compressed_pair.h"
```

Comment:
We have some common includes in `packages/Python/lldbsuite/test/make/`. I think it'd be better to put this there (probably in a subdirectory)

Reply:
Yup will do, this was the easiest for the prototype. But I'll move it to `packages/Python/lldbsuite/test/make/` as you suggest.